src/nexus-prisma/NexusPrismaBuilder.ts

```diff
@@ -229,31 +235,29 @@ export class NexusPrismaBuilder {
 
       if (!whereArg) {
         throw new Error(
-          `Could not find filtering argument for ${prismaModelName}.${field.name}. Searched for a field param with type "${whereTypeName}". Actual fields were ${field.args}`,
-        )
-      }
-
-      if (opts.filtering !== true) {
-        this.whitelistMap[
-          this.argTypeName(
-            graphQLTypeName,
-            field.name,
-            whereArg.inputType.type,
-            whereArg.inputType.kind,
-          )
-        ] = Object.keys(opts.filtering).filter(
-          fieldName => (opts.filtering as any)[fieldName] === true,
+          `Could not find filtering argument in ${prismaModelName}.${
+            field.name
+          }(${field.args
+            .map(a => `${a.name}: ${a.inputType.type}`)
+            .join(', ')}): ${field.outputType.type}`,
```

Comment:
Bit hard to read this

Reply:
Agreed but not sure how to factor it better. It generates something like `Query.posts(first: Int, last: Int, before: String, after: String)`

Reply:
I reverted it anyway, we'll think about better errors in another pr